views/uploadpackage.etlua

```diff
@@ -0,0 +1,54 @@
+<!-- 
+Help me, my creator knows nothing of html or css
+I suffer every time I look in the mirror
+Please end my pain.
+-->
+<form action="<%= burl("uploadpackage") %>" method="post" enctype="multipart/form-data">
+    <%= i18n("upload_button_header") %>
+    <br>
+    <input type="button" id="choosePackageFile" value="<%= i18n("choose_file") %>" onclick="document.getElementById('file').click();"/>
+    <input type="file" name="file" id="file" accept=".mpackage,.xml" style="display:none;" />
```

Comment:
`.zip` as well

Reply:
see above comment regarding zip files. Also, just for trivia's sake, this only effects what files the browser presents by default and can be heavily swayed by browser behaviour. So this is more of a strongly worded suggestion than anything